tango/integrations/wandb/step_cache.py

```diff
@@ -120,6 +121,7 @@ def get_step_result_artifact_url(self, step: Union[Step, StepInfo]) -> str:
             f"/{self._step_artifact_name(step)}/{step.unique_id}"
         )
 
+    @retry(RuntimeError, delay=10, backoff=2, max_delay=120)
```

Comment:
We don't want to retry for a `RuntimeError`, we probably only want to retry for `wandb.errors.CommError`.

```suggestion
    @retry(exceptions=(wandb.errors.CommError,), delay=10, backoff=2, max_delay=120)
```